.circleci/config.yml

```diff
@@ -190,17 +211,12 @@ jobs:
           condition:
             equal: [ "integreat", << parameters.deliver_to >> ]
           steps:
-            - run:
-                name: Read version number
-                command: |
-                  echo "export VERSION_NAME=$(cat version.json | jq .versionName)" >> $BASH_ENV
-                  source $BASH_ENV
             - run:
                 name: Integreat production delivery # StrictHostKeyChecking=no is not a security problem. The worst that could happen is a delivery to the wrong domain.
                 command: sftp -v -o StrictHostKeyChecking=no web@web.integreat-app.de:/integreat.app \<<< $'put -r /tmp/output/dist/*'
             - slack/notify:
-                message: Integreat WebApp version ${VERSION_NAME} was delivered successfully to integreat.app!
-                webhook: $SLACK_URL
+                message: Integreat WebApp version $NEW_VERSION_NAME was delivered successfully to integreat.app!
```

Comment:
Are you sure that the $NEW_VERSION_NAME gets resolved properly? The `prepare-integreat-delivery` job adds it to the BASH_ENV. But this bash env is not available here because it is run in another docker image

Reply:
It is available, I tested it.